### PR TITLE
 LPS-148113

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-148113

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please check if these failures are related to your changes? Thanks

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/servlet/taglib/definition/DocumentLibraryDataLayoutBuilderDefinition.java [hunk 1]
```diff
@@ -46,9 +46,9 @@ public boolean allowMultiplePages() {
 	@Override
 	public String[] getUnimplementedProperties() {
 		return new String[] {
-			"allowGuestUsers", "fieldNamespace", "hideField", "inputMask",
-			"readOnly", "requireConfirmation", "validation",
-			"visibilityExpression"
+			"allowGuestUsers", "confirmationErrorMessage", "confirmationLabel",
+			"direction", "fieldNamespace", "hideField", "inputMask", "readOnly",
+			"requireConfirmation", "validation", "visibilityExpression"
```
[the remark this answers is not in the document]
Comment (alinedoleron) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/servlet/taglib/definition/DocumentLibraryDataLayoutBuilderDefinition.java:51]: Thanks for the analysis, @carolmariaabb! I'll talk to you tomorrow =)

[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/servlet/taglib/definition/DocumentLibraryDataLayoutBuilderDefinition.java:51]: From my analysis, this bug was caused by this [commit](https://github.com/liferay/liferay-portal/commit/9df9742b6af5f9d40ce686e67e067e83a9a25e70).

**Comment by carolmariaabb** [comment]
Resent https://github.com/liferay-forms/liferay-portal/pull/1486

**Pull request closed without merging**